packages/manager/modules/pci/src/projects/project/instances/instance/instance.html

```diff
@@ -63,7 +63,7 @@
                 </oui-tile-button>
                 <oui-tile-button
                     data-on-click="$ctrl.scheduleAutoBackup($ctrl.instance)"
-                    data-ng-if="$ctrl.instance.statusGroup === 'ACTIVE' && !$ctrl.coreConfig.isRegion('US')"
+                    data-ng-if="$ctrl.instance.statusGroup === 'ACTIVE' && $ctrl.PciProjectsProjectInstanceService.automatedBackupIsAvailable($ctrl.instance.flavor.type)"
```

Comment:
**question:** Are we enabling this in `US` because I see the check `!$ctrl.coreConfig.isRegion('US')` has been removed.